heat/core/logical.py

```diff
@@ -307,3 +314,55 @@ def logical_xor(t1, t2):
     tensor([ False, False,  True])
     """
     return operations.__binary_op(torch.logical_xor, t1, t2)
+
+
+def _sanitize_close_input(x, y):
```

Comment:
Use double leading underscores for internal functions (yes we might have violated this in the past)

Reply:
Done thanks